src/components/navBar/navBar.module.scss

```diff
@@ -1,35 +1,137 @@
-$offset: 10px;
-$thickness: 3px;
+$font-stack: "Roboto", sans-serif;
+$color-white: #ffffff;
+$color-primary: #1d1283;
+$color-highlight: #49a82e;
+$color-highlight2: #e30464;
+$color-boxshadow: rgba(0, 0, 0, 0.5);
 
 .navBar {
+  font: 700 16px $font-stack;
```

Comment:
any reason we are not following `rem` for font?

Reply:
Here if I am using 1 rem which is equal to 16px , is not coming actually 16px, it's 15px on using it here. Not sure why it's happening but will try to make it `rem`

Reply:
Made the change